go/sig/internal/cfgmgmt/cfg.go

```diff
@@ -0,0 +1,90 @@
+package cfgmgmt
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"time"
+
+	"github.com/scionproto/scion/go/lib/log"
+	"github.com/scionproto/scion/go/sig/internal/mscomm"
+	"github.com/scionproto/scion/go/sig/internal/sigcmn"
+	"github.com/scionproto/scion/go/sig/internal/sigcrypto"
+	"github.com/scionproto/scion/go/sig/internal/sqlite"
+)
+
+var (
+	prefixFile string
+)
+
+//Init intiialises the package and also starts a poll on the file in prefixFilePath to push
```

Comment:
```suggestion
//Init initialises the package and also starts a poll on the file in prefixFilePath to push
```

Guess this has nothing to do with https://en.wikipedia.org/wiki/Inti ;-)